docs/02-app/01-building-your-application/04-caching/02-router-cache.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Router Cache
+description: Learn how the Router Cache works in Next.js.
+---
+
+The Next.js App Router uses an in-memory client-side cache that stores the rendered result of Server Components, split by invidiual route segments. This **Router Cache** is used to improve navigation performance by only [partially rendering]() the content that changes (instead of the whole route) and prefetching content that the user is likely going to navigate to. It stores both visited segments and prefetched segments.
+
+For the user, this results in [soft navigation]() (no page reloads) and faster page transitions; it also reduces unnecessary requests to the server.
+
+> **Client-side Cache and Prefetch Cache**
+>
+> You may see this cache being referred to as **Client-side Cache** or **Prefetch Cache**. While **Prefetch Cache** refers to the prefetched segments, **Client-side Cache** refers to the whole Router cache, which includes both visited and prefetched segments.
+> This cache specifically applies to Next.js and Server Components, and is different to the browser's [bfcache](https://web.dev/bfcache/).
+
+## How it works
+
+When the user first visits a route, Next.js will cache the rendered result of the route segments (layouts and page). If the route contains `<Link>` components (or uses `router.prefetch`), Next.js will automatically start prefetching and caching the content of those linked routes.
+
+As users navigate between routes, Next.js will check if the route segments are already cached. If so, it will skip sending a new request to the server for rendering, and use the payload of cached segments instead. This works when navigating to new routes, as well as backwards and forwards browser navigation.
+
+> **What is the rendered result?**
+>
+> You may see the terms like **RSC Payload**, **Rendered Result**, and **Flight Data** used interchangeably. They all refer to the same thing: the rendered result of Server Components, which is sent to the client. To learn more, see the [Server Components]() documentation.
+
+## Duration and Invalidation
+
+The cache is stored in the browser's temporary memory. There are two factors that determine how long the router cache lasts:
+
+- **Session**: The cache persists across soft navigation. However, it's cleared on page refresh or [hard navigation]().
+- **Automatic invalidation period**: The cache of an individual segment is automatically invalidated after a time period. The duration depends on whether the route is [static]() or [dynamic]():
+  - **Dynamic routes**: 30 seconds
```

Comment:
The term "dynamic routes" may be confusing, please refer to #52512 & #52545

Reply:
Thanks, I'd like to eventually call it **dynamic route segments**. 
However, I need to verify if this behaviour has been implemented. In other references in the docs, it's purposely called routes instead of route segments because the behaviour applied to the whole route, the goal was to eventually reach more granularity with route segments.